pkg/konfluxgen/konfluxgen.go

```diff
@@ -36,6 +36,9 @@ var PipelineDockerBuildTemplate embed.FS
 //go:embed fbc-builder.yaml
 var PipelineFBCBuildTemplate embed.FS
 
+//go:embed integration-test-scenario.template.yaml
+var IntegrationTestScenarioTemplate embed.FS
```

Comment:
I'd rename this one (file, variable, etc) along the lines of "EnterpriseContractTestScenarioTemplate"

Reply:
Since we might probably have more test scenarios in the future